EIPS/eip-5725.md

```diff
@@ -0,0 +1,181 @@
+---
+eip: 5725
+title: Transferable Vesting NFT
+description: An interface for transferable vesting NFTs which release underlying tokens over time.
+author: Apeguru (@Apegurus), Marco De Vries <marco@paladinsec.co>, Mario <mario@paladinsec.co>, DeFiFoFum (@DeFiFoFum)
+discussions-to: https://ethereum-magicians.org/t/eip-5725-transferable-vesting-nft/11099
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-09-08
+requires: 721
+---
+
+## Abstract
+A **Non-Fungible Token** (NFT) standard used to vest tokens ([EIP-20](./eip-20.md) or otherwise) over a vesting release curve. 
+
+The following standard allows for the implementation of a standard API for NFT based contracts that hold and represent the vested and locked properties of any underlying token ([EIP-20](./eip-20.md) or otherwise) that is emitted to the NFT holder. This standard is an extension of the [EIP-721](./eip-721.md) token that provides basic functionality for creating vesting NFTs, claiming the tokens and reading vesting curve properties.
```

Comment:
Thanks for authoring! Before anything else, can I ask a high level question? Do you intend to let this EIP support vesting ERC20s that are not in the same instance of the compliant contract?

Reply:
Yes we do! 

We actually have a live use case that is implemented in a way were the underlying ERC20 is not deposited in the same instance of the compliant contract. 

This implementation allows for such use case.